src/base64_rfc2045.ml

```diff
@@ -186,9 +141,9 @@ and t_flush {quantum; size; buffer} =
       unsafe_set_chr buffer 0 (unsafe_chr ((quantum lsr 8) land 255)) ;
       unsafe_set_chr buffer 1 (unsafe_chr (quantum land 255)) ;
       `Flush {quantum; size; buffer= Bytes.sub buffer 0 2}
-  | _ -> malformed buffer 0 0 3
+  | _ -> assert false
```

Comment:
At least, a comment to explain why this case is not occur is needed here.

Reply:
My bad, fixed now.
This can never happen as ``size`` will only ever be in the range``[0..3]``.